examples/timezone-aware.md

```diff
@@ -73,8 +73,8 @@ $('#clock').countdown(nextYear.toDate(), function(event) {
 
 ## Feature read
 
-- [MomentJS Documentation](http://momentjs.com/docs/)
-- [Moment Timezone Documentation](http://momentjs.com/timezone/docs/)
+- [MomentJS Documentation](https://momentjs.com/docs/)
```

Comment:
Ah, you're right! I thought I had checked all the links I replaced, but this one does not seem to work via HTTPS. I'll change it back (as these do not cause mixed content anyway).